### PR TITLE
Fixes #778: Remove Gaia

### LINKED ISSUE #778: Remove Gaia from supported products
It's time to say goodbye to Gaia and clean up after it.

### PR DESCRIPTION
**Pull request by TheoChevalier**

### CODE REVIEW

**Comment by flodolo** [comment]
Looks like there are more tests to cleanup/update (Health.php)?

**Comment by flodolo** [comment]
One more reference here
https://github.com/TheoChevalier/transvision/blob/227df97ebc414e9221115a0a36f3af670083acfe/app/classes/Transvision/Health.php#L208

Also this test has a lot of obsolete test-cases, since we remove Gaia's checks
https://github.com/TheoChevalier/transvision/blob/227df97ebc414e9221115a0a36f3af670083acfe/tests/units/Transvision/AnalyseStrings.php

**Comment by TheoChevalier** [comment]
I think they aren’t obsolete (yet) https://transvision.mozfr.org/?recherche=%7B%7B&repo=central&sourcelocale=en-US&locale=fr&search_type=strings Switched to desktop since it’s what makes most sense currently.

File: app/config/config.ini-travis [hunk 1]
```diff
@@ -12,7 +12,7 @@ install=/home/travis/build/mozfr/transvision/
 config=/home/travis/build/mozfr/transvision/app/config
 
 ; URL to l10n web service. Wrap the value between quotes if the URL contains special characters like ~
-l10nwebservice = "https://l10n.mozilla-community.org/~flod/mozilla-l10n-query/"
+l10nwebservice = "https://flod.org/mozilla-l10n-query/"
```
Comment (flodolo) [inline app/config/config.ini-travis]: Note: remember to restore the value, and also to drop the ~flod/ part

**Comment by flodolo** [comment]
Everything looks great to me. Should we merge and open a pull requests on-hold to restore the URL?

**Comment by TheoChevalier** [comment]
yeah, lets :)

**Pull request merged**